_projects/2022/100348375/100348375.Rmd

```diff
@@ -0,0 +1,981 @@
+opinion article written by columnist David Leonhardt. This journalist explained 
+how in recent years monopolies have increased in the United States despite the 
+reluctance of many Americans who consider them a threat to market competition 
+and individual freedom. This article is accompanied by a graph showing how the 
+combined market share has changed for the top two companies in many industries.
+
+***[The Monopolization of America](https://www.nytimes.com/2018/11/25/opinion/monopolies-in-the-us.html?unlocked_article_code=AAAAAAAAAAAAAAAACEIPuonUktbco4hlSVUZBibSRdkhrxqAwv3IxrskgXv_ITPQSDAH0-QYRpWbvUfRYK4pL_0C4wOmfv4Terk-WK41ieZFI01mTwTvqMqKyMUPfS5woo_pAiZrj5-XGLkzrmevP2Xidr5xlePjskrbenCqDPKEqQApJAlipZFgfFmtyCZIkv-DSrgpr4E4ifQxBZl6RiMDZD2IuZTrDRZ4O9aBa3LM-1V8GrEZCXyIw4nqu_9Xex5SCFnGUHt8_W0_jdtZM94XN6r5RAUyt9cDVL2W_2EO8RB4R-XivQ&smid=nytcore-ios-share&referringSource=articleShare)***
+
+
+```{r echo=FALSE, fig.cap="Source: New York Times", out.width = '100%'}
+knitr::include_graphics("market_share.png")
+```
+
+
+In this tutorial I will show how to replicate this data visualization step by 
+step with R and the package ggplot2.
+
+```{r}
+# Load libraries:
+library(tidyverse)
+library(ggplot2)
+```
+
+
+### **1-Read and prepare the data**
+
+Since the chart's data source was not directly accessible, I created a csv file 
+with the data. This file contains data for three different variables: the type 
+of industry, the market share in the year 2000, and the current market share. 
+Also, since in the original graph the positive and negative increases over time 
+in the market share are differentiated, I create a new variable (*increase*) 
+with this information.
+
+```{r}
+
+data <- read.csv("data_graph.csv", sep = ";") %>% 
+  mutate(increase = market_share_now - market_share_2000,
+         increase2 = case_when(increase > 0 ~ "positive",
+                              T ~ "negative")) %>%
+  group_by(increase2) %>% 
+  arrange(desc(increase))
+
+```
+
+Let's see how the data is!
+
+```{r}
+
+glimpse(data)
+
+```
+
+``` {r}
+
+summary(data)
+
+```
+
+### **2-Create the first part of the plot**
+
+The chosen graph has two distinct parts. The first one shows the evolution of 
+the market share since the early 2000s (2002 to 2007 depending on data 
+availability) and the second since 2012. For this reason, I will do the upper 
+graph first and then the lower one.
+
+The first step, therefore, is to filter the data to use only those that 
+correspond to the industries included in the first graph. Also, I need to create
+a new variable that allows me to order the categorical variable "industry" from 
+the highest to lowest increase in market share over time (*industry_order*)
+
+``` {r}
+
+upper_plot <- data %>%
+  filter(industry != "Smartphones" & industry != "Social networks" 
+         & industry != "Cellphone service") %>% 
+  mutate(industry_order = fct_reorder(industry, increase)) 
+
+```
+
+### COORDINATES AND AXES
```

Comment:
This (and the ones below) should be a level 4 heading (`####`), because I understand that this is meant to be _inside_ "Create the first part of the plot", right? Also, please use sentence-style capitalization in all titles, just to follow a uniform style.